_data/security.yml

```diff
@@ -71,6 +71,13 @@ websites:
       hardware: Yes
       doc: http://www.entrust.net/certificate-services/cms-authentication.htm
 
+    - name: ExpresssVPN
+      img: expressvpn.png
+      url: https://www.expressvpn.com
+      tfa: No
+      twitter: ExpresssVPN
```

Comment:
Twitter has an extra "s" inserted and is not capitalized according to their Twitter username.
Line should be `twitter: expressvpn`